src/modules/displayList.js

```diff
@@ -0,0 +1,70 @@
+import retrieveFromStore from './retrieveFromStore.js';
+import selectItem from './selectItem.js';
+import deleteItem from './deleteItem.js';
+import listStore from './listStore.js';
+
+const displayList = () => {
+  const todoList = document.querySelector('.todo-list-items');
+  const welcomeMessageContainer = document.querySelector('.welcome-msg');
+  welcomeMessageContainer.innerHTML = '';
+  todoList.innerHTML = '';
+
+  retrieveFromStore().forEach((item) => {
+    const listItem = document.createElement('div');
+    listItem.classList.add('list-div');
+    const todoItem = document.createElement('input');
+    todoItem.type = 'text';
+    todoItem.classList.add('list-item');
+    todoItem.setAttribute('id', `input-${item.index}`);
+    const checkbox = document.createElement('input');
+    checkbox.classList.add('checkbox-item');
+    checkbox.type = 'checkbox';
+    checkbox.setAttribute('id', `${item.index}`);
```

Comment:
Well done on incorporating dynamic `id` attributes for each checkbox, which is a positive step. However, after reviewing the page, I noticed an area for improvement:

- [ ] [Optional] When an item has a `completed` property value of `true`, the corresponding list item should be checked upon reloading the page. I recommend enhancing your code to dynamically assign the status of the checkbox based on the `completed` property, similar to how you've implemented the dynamic `id` attribute. This adjustment will ensure that the checkboxes accurately reflect the completion status of the tasks.